### DISCUSSION TITLE
How to insert into database

### DISCUSSION THREAD

**Discussion by keithelder** [dylanljones/pyrekordbox#130] [Q&A] [answered]
I want to automate my workflow with this script if possible. I want to: 

1.  Drop some files in a folder
2. Run the script
3. Have it parse the files and copy them to folder structure; /folder/artist/file
4. Insert into the Rekordbox database as a playlist (so I can easily find them and catalog them)
5. Analyze the files. 

I didn't do a lot of digging with this script but that's the workflow I want to automate with Recordbox. Is that possible with pyrekordbox? Thanks. If so, how would I get started?

**Comment by jpawlik** [comment, marked as answer]
The Tutorials in the docs go over this.
>A new playlist can be created by calling the `db.create_playlist()` method ([source](https://pyrekordbox.readthedocs.io/en/latest/tutorial/db6.html#playlists))

>New tracks can be added to your rekordbox collection by calling the `db.add_content()` method. ([source](https://pyrekordbox.readthedocs.io/en/latest/tutorial/db6.html#tracks))
